src/snowflake/snowpark/_internal/ast.py

```diff
@@ -144,8 +144,9 @@ def flush(self) -> Tuple[str, str]:
     def _init_batch(self):
         self._request_id = uuid.uuid4()  # Generate a new unique ID.
         self._request = proto.Request()
-        self._request.client_version.major = 42
```

Comment:
should we remove this? what is the client_version here, the Snowpark client version (i.e., 1.20?)?

Reply:
I added a TODO here.